DabraHalleloEndaAbunaSamuel/DHAS009.xml

```diff
@@ -0,0 +1,128 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="DHAS009" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Miracles of St Mary</title>
```

Comment:
Do we need St.? If yes, probably we need to add St. to everyone considered a saint. But Miracles of Mary sounds more common for me.

Reply:
Not really.